autoflip_calculators/text_detection_calculator_test.cc

```diff
@@ -78,7 +78,9 @@ CalculatorGraphConfig::Node MakeConfig(std::string base_config, bool use_visual_
   config.mutable_options()
```

Comment:
I see you have kConfig and kConfigNoVideo.  Probably we don't need the NoVideo part, right?